packages/payments-portal/serverless.yml

```diff
@@ -0,0 +1,105 @@
+service: cloud-payments-portal
```

Comment:
It is possibly we could deploy this with CDK instead of serverless? Mitigates converting in the future

Reply:
Done